docs/app/pages/components/components-sections/search/toolbar-search/toolbar-search.component.html

```diff
@@ -68,8 +68,11 @@
 
 <p>
     Other components can be added as required. The toolbar search component on the left above shows how to include an
-    optional "clear" button to the input field using the built-in <code>ux-toolbar-search-clear</code> class. Another
-    common use-case is to include a <a href="https://valor-software.com/ngx-bootstrap/#/typeahead">typeahead</a> list to
+    optional "clear" button to the input field using the built-in <code>ux-toolbar-search-clear</code> class.
+    If the clear button is added, you should add <code>[class.ux-toolbar-search-clear-offset]="clickEvent"</code> to the
+    <code>Input</code> field element (with <code>"clickEvent"</code> being the name of the click event variable).
+    This will bind the <code>ux-toolbar-search-clear-offset</code> class to the click event and add the correct padding.
+    Another common use-case is to include a <a href="https://valor-software.com/ngx-bootstrap/#/typeahead">typeahead</a> list to
```

Comment:
Since the description of adding a clear button is a bit longer now, could you make this typeahead sentence into its own paragraph?